tests/components/light/test_zwave.py

```diff
@@ -10,7 +10,7 @@ def test_get_device_detects_dimmer(mock_openzwave):
     node = MockNode()
     value = MockValue(data=0, node=node)
 
-    device = zwave.get_device(node, value, {})
+    device = zwave.get_device(node=node, value=value, node_config={})
```

Comment:
As it turns out the binary_sensor get_device only includes these as keyword arguments, not positional, so I figured we should be consistent with our calls across all the tests where reasonable.